components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleErrorConstants.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.application.authenticator.google;
+
+/**
+ * This class defines all the error codes and messages which are specific for
+ * Google authenticator.
+ */
+public class GoogleErrorConstants {
+
+    public enum ErrorMessages {
+
+        CSRF_VALIDATION_FAILED_ERROR("60000", "CSRF cookie validation failed in Google one tap. " +
+                "Authenticator : " + GoogleOAuth2AuthenticationConstant.GOOGLE_CONNECTOR_NAME + " Client Id : %s ."),
```

Comment:
Wouldn't this be more readable if we add the error message in one sentence as follows?
```suggestion
        CSRF_VALIDATION_FAILED_ERROR("60000", "CSRF cookie validation failed in Google one tap " +
                "authenticator: " + GoogleOAuth2AuthenticationConstant.GOOGLE_CONNECTOR_NAME + " with client id: %s ."),
```

Reply:
Fixed 3335ebd823fbdae6364b8cc712db7975815b8ee6